theme/n21/views/authentication/home.hbs

```diff
@@ -12,12 +12,11 @@
 {{#content "body"}}
 {{#embed "lib/header"}}{{/embed}}
 <div class="container">
-    <div class="row">
+    <div class="row" style="flex-direction: row-reverse;">
```

Comment:
Same here, I would suggest outsourcing the css here. This makes refactoring etc. so much easier and complies with our rules. https://github.com/hpi-schul-cloud/schulcloud-client/blob/develop/docs/CSP_RULES_DEV.md